static/js/pages/instances/actions/snapshots/CreateSnapshotBtn.tsx

```diff
@@ -2,30 +2,27 @@ import React, { FC, useState } from "react";
 import { Button } from "@canonical/react-components";
 import CreateSnapshotForm from "./CreateSnapshotForm";
 import { LxdInstance } from "types/instance";
-import { NotificationHelper } from "types/notification";
+import useNotify from "util/useNotify";
 
 interface Props {
   instance: LxdInstance;
-  notify: NotificationHelper;
 }
 
-const CreateSnapshotBtn: FC<Props> = ({ instance, notify }) => {
+const CreateSnapshotBtn: FC<Props> = ({ instance }) => {
+  const notify = useNotify();
   const [isModal, setModal] = useState(false);
```

Comment:
You could probably that by naming it `[open, setOpen]` or `[modalOpen, setModalOpen]`

Reply:
yeah that is better.